apps/docs/src/components/ex/Timeline/style.css

```
@@ -41,17 +41,17 @@
     --_line-len: calc(100% - var(--_line-offset));
   }
 
-  .c--timeline--horizontal {
+  .b--timeline--horizontal {
     --edge: var(--s15);
     --_line-offset: calc(50% - var(--edge) / 2);
     grid-template-rows: var(--shapeSpace) auto auto auto;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)